### ISSUE TITLE
Feature: labgrid-client ssh to use NetworkService if available 

### ISSUE THREAD

**Issue by andreasdotzler** [labgrid-project/labgrid#287]
labgrid-client ssh currently requires labgrid.resource.base.EthernetPort available for the target. It should use NetworkService if available.

**Comment by jluebbe** [comment]
Yes, that seems to be missing from client.py. I'd be happy to merge a PR for this.

**Comment by BastianStender** [comment]
I implemented this with #342. Please test it.

**Comment by maximilianriemensberger** [comment]
@Bastian-Krause I tested #342. It down not work with the FormwardedNetworkService since the port is not 22 in this case.
